src/hooks/useConcepts.ts

```diff
@@ -28,7 +28,9 @@ export function useConcepts(references: Set<string>): {
     const start = index * chunkSize;
     const end = start + chunkSize;
     const referenceChunk = Array.from(references).slice(start, end);
-    return `${restBaseUrl}/concept?references=${referenceChunk.join(',')}&v=${conceptRepresentation}`;
+    return `${restBaseUrl}/concept?references=${referenceChunk.join(
```

Comment:
As defined in the PR description, the `chunkSize` must be fetched from the global properties, endpoint `${restBaseUrl}/systemsetting/<setting_name>`